docker/s6/services/.s6-svscan/crash

```diff
@@ -1,7 +1,3 @@
 #!/bin/sh
 
-echo "Container crashed"
-
-echo "Trying to clean file cache first..."
-
-rm -rf /lazyraster/cache
\ No newline at end of file
+echo "Container crashed"
```

Comment:
From my tests, it looks like `crash` is not called when `finish` is present.

According to the s6 docs https://skarnet.org/software/s6/s6-svscan.html:
> When it encounters a fatal situation, something it really cannot handle, it executes into .s6-svscan/crash instead of dying; when it is told to exit, it executes into .s6-svscan/finish. Administrators should make sure to design appropriate crash and finish routines.

I tried to dig a bit through the s6 code, but it's too magical to comprehend. It would be interesting to see if we'll ever find `Container crashed` in the logs if we have both a `finish` and a `crash` script in there, so I'd be happier to keep both of them.